client/version/updater.go

```diff
@@ -54,13 +54,28 @@ func (u *updater) Update() error {
 	}
 
 	err = u.unzip()
+	if err != nil {
```

Comment:
could use the `if err := u.unzip(); err != nil {` syntax instead. For this and for the following function calls.